bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableTableAdminClient.java

```diff
@@ -203,4 +211,49 @@ void waitForReplication(BigtableTableName tableName, long timeout)
   /** @deprecated Snapshots will be removed in the future */
   @Deprecated
   ListenableFuture<Operation> createTableFromSnapshotAsync(CreateTableFromSnapshotRequest request);
+
+  /**
+   * Creates a new backup from a table in a specific cluster.
+   *
+   * @param request a {@link CreateBackupRequest} object.
+   * @return The long running {@link Operation} for the request.
+   */
+  ListenableFuture<Operation> createBackupAsync(CreateBackupRequest request);
+
+  /**
+   * Gets metadata information about the specified backup.
+   *
+   * @param request a {@link GetBackupRequest} object.
+   * @return The {@link Backup} defined by the request.
+   */
+  ListenableFuture<Backup> getBackupAsync(GetBackupRequest request);
+
+  /**
+   * Updates the specified backup.
+   *
+   * @param request the request to update the Backup.
+   */
+  ListenableFuture<Backup> updateBackupAsync(UpdateBackupRequest request);
+
+  /**
+   * Lists all backups associated with the specified cluster.
+   *
+   * @param request a {@link GetBackupRequest} object.
+   * @return The {@link ListBackupsResponse} which has the list of the backups in the cluster.
+   */
+  ListenableFuture<ListBackupsResponse> listBackupsAsync(ListBackupsRequest request);
+
+  /**
+   * Permanently deletes the specified backup.
+   *
+   * @param request a {@link DeleteBackupRequest} object.
+   */
+  ListenableFuture<Empty> deleteBackupAsync(DeleteBackupRequest request);
+
+  /**
+   * Creates a new table from a backup.
+   *
+   * @param request a {@link DeleteBackupRequest} object.
```

Comment:
I think there needs to be some documentation as to what needs to happen next with the return type. As it stands users dont know what payload is serialized in it nor how they can find out when the restore operation actually completes.

I would recommend to hide the operation type altogether and just return ListenableFuture <RestoredTableResult> here

Reply:
This still needs a bit more doc